docs/shipping/Containers/control-plane.md

```diff
@@ -0,0 +1,87 @@
+---
+id: Control-plane
+title: Control Plane
+overview: Control Plane is a hybrid platform that integrates multiple cloud services, such as AWS, GCP, and Azure, providing a unified and flexible environment for developers to build and manage backend applications and services across various public and private clouds.
+product: ['logs']
+os: ['windows', 'linux']
+filters: ['Containers', 'Most Popular']
+logo: https://logzbucket.s3.eu-west-1.amazonaws.com/logz-docs/shipper-logos/control-plane.png
+logs_dashboards: []
+logs_alerts: []
+logs2metrics: []
+metrics_dashboards: []
+metrics_alerts: []
+drop_filter: []
+---
+
+## Logs
+
+Control Plane is a hybrid platform that integrates multiple cloud services, such as AWS, GCP, and Azure, providing a unified and flexible environment for developers to build and manage backend applications and services across various public and private clouds.
+
+Deploy this integration to ship all `Org` logs from your Control Plane account to Logz.io.
+
+**Before you begin, you'll need**:
+
+* Control Plane account
+
+
+
+### Store the Logz.io secret as an Opaque Secret
+
+1. Log in to [Control Plane Console UI](https://console.cpln.io/).
+2. Navigate to **Secrets**.
+3. Create a new secret.
+4. Give the secret a name.
+5. Select `Opaque` from the **Secret Type list**.
+6. Paste `<<LOG-SHIPPING-TOKEN>>` into the content text box.
+7. Click **Create**.
+
+{@include: ../../_include/log-shipping/log-shipping-token.html}
```

Comment:
The placeholder will probably look ok in the app, but it looks a little funny in the public docs, because it looks just like this <<LOG-SHIPPING-TOKEN>> without any explanation that this is a logzio log shipping token.